app_power_module/src/telemetry.c

```diff
@@ -27,8 +27,8 @@ K_THREAD_DEFINE(ina_thread, SENSOR_READ_STACK_SIZE, ina_task, NULL, NULL, NULL,
 K_THREAD_DEFINE(adc_thread, SENSOR_READ_STACK_SIZE, adc_task, NULL, NULL, NULL, K_PRIO_PREEMPT(10), 0, 1000);
 
 // Message Queues
-K_MSGQ_DEFINE(ina_telemetry_msgq, sizeof(power_module_telemetry_t), 10, 4);
-K_MSGQ_DEFINE(adc_telemetry_msgq, sizeof(float), 20, 4);
```

Comment:
Bring this back. We don't want/need to send timestamps over Ethernet. Its just for data logging purposes. You can just pass in the timed_telemetry.data field and the sizeof(power_module_telemetry_t) here for example